browser/widevine/widevine_permission_request_browsertest.cc

```diff
@@ -258,8 +258,16 @@ class ScriptTriggerWidevinePermissionRequestBrowserTest
   net::EmbeddedTestServer https_server_;
 };
 
+#if defined(OFFICIAL_BUILD) && BUILDFLAG(IS_WIN)
+#define MAYBE_SuggestPermissionIfWidevineDetected \
+  DISABLED_SuggestPermissionIfWidevineDetected
+#else
+#define MAYBE_SuggestPermissionIfWidevineDetected \
+  SuggestPermissionIfWidevineDetected
+#endif
+
```

Comment:
nit: usually there is no empty line between `MAYBE_` defines and the test itself.

Reply:
Updated.